scripts/generate-i18n-messages.ts

```diff
@@ -37,7 +37,9 @@ const logger = getLogger({ isSilent });
         const baseThemeDirPath = pathJoin(tmpDirPath, "base");
 
         crawl(baseThemeDirPath).forEach(filePath => {
-            const match = filePath.match(/^([^/]+)\/messages\/messages_([^.]+)\.properties$/);
+            const match =
+                filePath.match(/^([^/]+)\/messages\/messages_([^.]+)\.properties$/) ||
+                filePath.match(/^([^\\]+)\\messages\\messages_([^.]+)\.properties$/);
```

Comment:
It woule be cleaner to use `new Regexp(`([^\\${path.sep}]+)\/messages\\${path.sep}messages_([^.]+)\.properties$`)`) but okay.